src/components/render-settings-panel.js

```diff
@@ -65,13 +77,129 @@ function sceneBuilder(animationLoop) {
   return new DeckScene({
     animationLoop,
     keyframes,
-    lengthMs: 5000,
+    lengthMs: 500,
     data,
    width: 480,
    height: 460
   });
 }
 
+function setKeyframes(cameraType){
+  adapter.scene.keyframes.camera._lastTime = 0;
+  adapter.scene.keyframes.camera.factor = 0;
```

Comment:
Awesome that you found something that works. I see something missing in hubble to do this the "intended way"

```
adapter.scene.animationLoop.timeline.setTime(0)
if(cameraType === 'Orbit (90º)'){
  const handle = this.state.cameraHandle; // Hubble should help maintain this, since it increases every time a new animation is attached. See: https://github.com/visgl/luma.gl/blob/master/modules/engine/src/animation/timeline.js
  
  adapter.scene.animationLoop.timeline.detachAnimation(handle)
  const camera = new CameraKeyframes({	
        timings: [0, 2000],	
        keyframes: [	
          {	
            longitude: 0,	
            latitude: 11,	
            zoom: 2,	
            pitch: 0,	
            bearing: 0	
          },	
          {	
            longitude: 0,	
            latitude: 11,	
            zoom: 2,	
            bearing: 90,	
            pitch: 0	
          }	
        ],	
        easings: [easing.easeInOut]	
      })	
    };	
    const newCameraHandle = adapter.scene.animationLoop.timeline.attachAnimation(camera);
    this.setState({cameraHandle: newCameraHandle})
}
```